websrc/source/documents/tutorials/contiv-policy.md

```diff
@@ -42,6 +42,34 @@ make demo-swarm
 This will create two VMs on VirtualBox. Using ansible, all the required services and software for contiv, will get installed at this step.
 This might take some time (usually approx 15-20 mins) depending upon your internet connection.
 
+-- or --
+#### Step 2a: Create a vagrant VM cluster
+
+```
+make cluster
+```
+
+This will create two VMs on VirtualBox. It will also create a .cfg.yml config file that will be used in later steps. Also setup the following two config vars
+
+```
+cd cluster
```

Comment:
from here it should be 2a,2b,etc. because this needs to be done irrespective of `demo-swarm/cluster`?